content/en/docs/contribute/_index.md

```diff
@@ -31,6 +31,10 @@ Kubernetes documentation contributors:
 
 <!-- body -->
 
+<br>
+
+---
```

Comment:
Hi @chrismetz09
Can you check on lines 34-36? Why are these lines included on the page?

Reply:
Hey @kbhawkey, I used for line break and horizontal line between the sections on this page. I figured it tightened appearance and makes for easier reading.

Reply:
Thanks for the explanation!
I suggest to remove the horizontal line between sections. The doc pages do not use these element between headings.
I like the diagrams. Is it possible to increase the font size in the first diagram? I find the text hard to read. Thanks.

Reply:
will remove "---". I Re: font-size I tested increases from font-size:15px. We  can go to 16px before text is cropped inside. This is an isssue with Mermaid and recommended fixes involve possibly changing  CSS. I would prefer to open separate PR to handle that. Would that be okay? I want to get this in so we can point to docs url in kubecon sig docs deck. Would that be okay? Tx @kbhawkey for the review.

Reply:
That sounds okay @chrismetz09. Thanks.